src/components/Filter/FilterRemove.js

```diff
@@ -0,0 +1,31 @@
+import cx from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FilterRemove = ({ className, onFilterRemoveClick, title, ...rest }) => {
+  const classes = cx('pficon pficon-close', className);
```

Comment:
we voted to start using `classNames` in the PF React Community meeting (was a live survey)  ;)

I will open a separate issue on this now...